src/components/PupilViews/PupilLessonOverview/PupilLessonOverview.view.tsx

```diff
@@ -1,67 +1,201 @@
+    }
+
+    if (sectionResults[section]) {
+      return "in-progress";
+    }
+
+    return "not-started";
+  }
 
   return (
-    <OakFlex
-      $pa={"inner-padding-l"}
-      $flexDirection={"column"}
-      $alignItems={"start"}
-      $gap={"space-between-s"}
+    <OakLessonLayout
+      lessonSectionName={"overview"}
+      topNavSlot={null}
+      bottomNavSlot={
+        <OakLessonBottomNav>
+          <OakPrimaryButton
+            onClick={proceedToNextSection}
+            width={["100%", "auto", "auto"]}
+            iconName="arrow-right"
+            isTrailingIcon
+          >
+            Let's get ready
+          </OakPrimaryButton>
+        </OakLessonBottomNav>
+      }
     >
-      <OakHeading tag="h1">Overview</OakHeading>
-      <OakUL>
-        {lessonSections
-          .filter((s) => s !== "overview" && s !== "review")
-          .map((s) => (
-            <OakLI
-              key={s}
-              $mv="space-between-ssx"
-              $pa={"inner-padding-s"}
-              $ba="border-solid-s"
+      <OakGrid
+        $maxWidth={["100%", "all-spacing-23", "100%"]}
+        $mt="space-between-s"
+        $mb={["space-between-none", "space-between-s"]}
+        $mh="auto"
+        $ph={["inner-padding-s", "inner-padding-xl", "inner-padding-none"]}
+      >
+        <OakGridArea $colStart={[1, 1, 2]} $colSpan={[12, 12, 10]}>
+          <OakTertiaryButton disabled iconName="arrow-left">
+            View all lessons
+          </OakTertiaryButton>
+        </OakGridArea>
+      </OakGrid>
+      <OakFlex
+        $minHeight="100%"
+        $alignItems={["flex-start", "flex-start", "center"]}
+        $pv="inner-padding-xl"
+        $ph={["inner-padding-none", "inner-padding-xl", "inner-padding-none"]}
+        $width="100%"
+        $maxWidth={["100%", "all-spacing-23", "100%"]}
+        $mh="auto"
+      >
+        <OakGrid $cg="space-between-s">
+          <OakGridArea
+            $colStart={[1, 1, 2]}
+            $colSpan={[12, 12, 5]}
+            $mb={["space-between-xl", "space-between-xl", "space-between-none"]}
+          >
+            <OakFlex
+              $flexDirection={["row", "row", "column"]}
+              $alignItems={["center", "center", "flex-start"]}
+              $gap={[
+                "space-between-s",
+                "space-between-m",
+                "space-between-none",
+              ]}
+              $borderColor="bg-decorative1-main"
+              $pb={["inner-padding-l", "inner-padding-none"]}
+              $ph={["inner-padding-s", "inner-padding-none"]}
+              $bb={["border-solid-l", "border-solid-none", "border-solid-none"]}
             >
-              <OakFlex
-                $gap={"space-between-ssx"}
-                $justifyContent={"space-between"}
-              >
-                <OakSecondaryButton
-                  onClick={() => {
-                    updateCurrentSection(s);
-                  }}
+              {isValidIconName(subjectIconName) && (
```

Comment:
The GQL type for `subjectSlug` is wide so this guards against values that we're not expecting. If we don't have an icon for the subject we won't display one.